components/elm/src/data_types/GridcellType.F90

```diff
@@ -45,10 +46,15 @@ module GridcellType
      integer , pointer :: pftf         (:) => null() ! ending pft index for each gridcell
      integer , pointer :: npfts        (:) => null() ! number of patches for each gridcell
 
-     ! Daylength
-     real(r8) , pointer :: max_dayl    (:) => null() ! maximum daylength for this grid cell (s)
+     ! Physical properties
+     real(r8) , pointer :: max_dayl    (:) => null() ! maximum daylength for this grid cell (seconds)
      real(r8) , pointer :: dayl        (:) => null() ! daylength (seconds)
      real(r8) , pointer :: prev_dayl   (:) => null() ! daylength from previous timestep (seconds)
+     real(r8) , pointer :: elevation   (:) => null() ! mean soil surface elevation, above mean sea level (m)
+     real(r8) , pointer :: froudenum   (:) => null() ! Froude number (dimensionless)
+	 real(r8) , pointer :: MaxElevation   (:) => null() ! Maximum soil surface elevation, above mean sea level (meter) needed for precipitation downscaling
```

Comment:
Fix indentation at lines 55, 113, 151

Reply:
@thorntonpe I have fixed indentation problems in my latest commit. Thanks.